lib/msf/ui/console/command_dispatcher/db.rb

```diff
@@ -1721,75 +1702,129 @@ def cmd_db_status(*args)
     return if not db_check_driver
 
     if framework.db.connection_established?
-      cdb = ''
-      ::ActiveRecord::Base.connection_pool.with_connection do |conn|
-        if conn.respond_to?(:current_database)
-          cdb = conn.current_database
-        end
-      end
-      print_status("#{framework.db.driver} connected to #{cdb}")
+      print_connection_info
     else
       print_status("#{framework.db.driver} selected, no connection")
     end
   end
 
   def cmd_db_connect_help
-    # Help is specific to each driver
-    cmd_db_connect("-h")
+    print_line("   Usage: db_connect <user:pass>@<host:port>/<database>")
+    print_line("      OR: db_connect -y [path/to/database.yml]")
+    print_line("      OR: db_connect [options] <http|https>://<host:port>")
+    print_line("Examples:")
+    print_line("       db_connect user@metasploit3")
+    print_line("       db_connect user:pass@192.168.0.2/metasploit3")
+    print_line("       db_connect user:pass@192.168.0.2:1500/metasploit3")
+    print_line("       db_connect http://localhost:8080")
+    print_line("       db_connect -c ~/cert.pem -t 6a7a74c1a5003802c955ead1bbddd4ab1b05a7f2940b4732d34bfc555bc6e1c5d7611a497b29e8f0 https://localhost:8080")
+    print_line(" ")
+    print_line("   OPTIONS:")
+    print_line("       -l,--list-services List the available data services that have been previously saved.")
+    print_line("       -y,--yaml          Connect to the database specified in the provided database.yml file.")
+    print_line("       -n,--name          Specify a name to save this connection as. Providing the name of a connection that already exists will overwrite existing connection.")
```

Comment:
Minor: consider something along the lines of "Name used to store the connection. Providing an existing name will overwrite the settings for the connection."

Reply:
I had pretty much that same text and ended up changing it to this. I'll change it back. Should have stuck with my gut.